Task_3/FileManagement.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_3
+{
+    public class FileManagement
+    {
+        public void ReadData()
+        {
+            string bufferAll = File.ReadAllText(Config.FullPath);
+            string[] bufferWorlds = bufferAll.Split(new char[] { ' ', '\n' }, StringSplitOptions.RemoveEmptyEntries);
+            string[] bufferLines = File.ReadAllLines(Config.FullPath);
```

Comment:
This approach reads the same file twice and this could lead to performance leaks.
You should have been splitted buffer all in two operations and count lines at first and then count words.